example/lib/main.dart

```diff
@@ -178,6 +189,22 @@ class PhoneFormFieldScreenState extends State<PhoneFormFieldScreen> {
                           setState(() => isCountryChipPersistent = v),
                       title: const Text('Persistent country chip'),
                     ),
+                    SwitchListTile(
+                      value: showDialCode,
+                      onChanged: (v) => setState(() => showDialCode = v),
+                      title: const Text('Show Dial Code'),
+                    ),
+                    SwitchListTile(
```

Comment:
This one can be removed

Reply:
Show dial code 
When Australia +61
Will not diplay when it false

Reply:
what is  main example ?

Reply:
the only example

Reply:
Ok i will remove it

Reply:
The problem is that when running the example on some phone  the input goes below the screen when there is too many options

Reply:
ok i return back to original code removed any change 
thank you for your time